libs/stack/stack-ui/src/components/Alerts/components/pagination/AlertsPaginationBullet.tsx

```diff
@@ -0,0 +1,33 @@
+import useThemeContext from 'libs/stack/stack-ui/src/providers/Theme/hooks'
+import type { TAlertsPaginationBulletProps } from '../../interface'
+
+const AlertsPaginationBullet = (props: TAlertsPaginationBulletProps) => {
+  const { themeName, tokens, controller, alerts, activeIndex, index } = props
+
+  const paginationBulletTheme = useThemeContext(`${themeName}.bullet`, tokens)
+  const paginationActiveBulletTheme = useThemeContext(`${themeName}.activeBullet`, tokens)
+  const alert = alerts[index]
+  const isActive = index === activeIndex
+
+  return (
+    <span
+      tabIndex={0}
+      role="button"
+      onKeyDown={(event) => {
+        if (event.key === 'Enter' || event.key === ' ') {
+          event.preventDefault()
+          controller?.slideTo(index)
+        }
+      }}
+      onClick={() => controller?.slideTo(index)}
+      key={JSON.stringify(alert)}
+      className={`${paginationBulletTheme} ${isActive ? paginationActiveBulletTheme : ''}`}
+      aria-current={isActive ? 'true' : 'false'}
+      aria-label={`${index + 1} of ${alerts.length}`}
```

Comment:
- aria-labelledby devrait pointer vers l'id de l'élément auquel il fait référence, pas le texte directement 
- aria-labelledby devrait seulement se faire set s'il y a un titre (testé avec le screen reader, il priorise aria-labelledby au-dessus de aria-label s'il y a les deux)
- Le aria-label ne devrait pas être hard codé en anglais. Pour ça on pourrait peut-etre utiliser [radash template](https://radash-docs.vercel.app/docs/string/template) pour avoir accès a l'index si on lui envoie une traduction

https://tink.uk/the-difference-between-aria-label-and-aria-labelledby/

Reply:
J'ai fix le issue de aria-labelledby en ajoutant un id sur le wrapper pis en faisant reference au id. Pour la traduction jai remplacé le `of` par un `/` comme ça pas besoin de traduire